context/app/api/vitessce_confs/base_confs.py

```diff
@@ -269,20 +272,27 @@ def __init__(self, entity, nexus_token, is_mock=False, **kwargs):
         # All "file" Vitessce objects that do not have wrappers.
         super().__init__(entity, nexus_token, is_mock)
         self._base_name = kwargs["base_name"]
+        self._mask_name = kwargs["mask_name"]
+        self._image_name = kwargs["image_name"]
         self._imaging_path = f"{self.image_pyramid_regex}/{kwargs['imaging_path']}"
+        self._mask_path = f"{self.image_pyramid_regex}/{kwargs['mask_path']}"
+
+    def _make_bitmask_image_path(self):
```

Comment:
```suggestion
    def _get_bitmask_image_path(self):
```

Or maybe save this for a future PR, if it's a lot of changes.